packages/scss/src/main.dist.scss

```diff
@@ -1,4 +1,3 @@
-@use 'commons';
-@use 'components'; // chargement de tout les composants
-
-@import '../../ng/style/main';
+@use 'commons'; // Common variables
+@use 'components'; // SCSS components
+//@use '../../ng/style/main'; // NG style
```

Comment:
we do not keep it ?

Reply:
It is moved in the configuration of storybook.

Reply:
@GuillaumeNury It was initially added in _Next_ to call ng style in scss dist instead of client side. (Then we forget 😬)
(I'll delete the commented line on a further PR)